book/src/languages.md

```diff
@@ -61,7 +61,8 @@ These configuration keys are available:
 | `roots`               | A set of marker files to look for when trying to find the workspace root. For example `Cargo.lock`, `yarn.lock` |
 | `auto-format`         | Whether to autoformat this language when saving               |
 | `diagnostic-severity` | Minimal severity of diagnostic for it to be displayed. (Allowed values: `Error`, `Warning`, `Info`, `Hint`) |
-| `comment-token`       | The token to use as a comment-token                           |
+| `comment-tokens`      | The tokens to use as a comment token, either a single token `"//"` or an array `["//", "///", "//!"]` (the first token will be used for commenting) |
```

Comment:
While it doesn't sound as niceas the plural version  I would prefer we keep `comment-token` to avoid a breaking change.

Reply:
Ah actually you can keep the change and use `serde(alias = 'comment-token')` yo keep legacy configs working